tests/Application/public/.htaccess

```diff
@@ -0,0 +1,25 @@
+DirectoryIndex app.php
```

Comment:
Do we need this file?

Reply:
I guess that's something from the old SF3.4 times. It should be changed to `index.php` I guess